modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala

```diff
@@ -0,0 +1,38 @@
+package com.dimafeng.testcontainers
+
+import org.testcontainers.containers.{ClickHouseContainer => JavaClickHouseContainer}
+
+class ClickHouseContainer(
```

Comment:
I think we need to have all classes declared as case classes for consistency

Reply:
Done